File: backend/DataAccess/DaoGenerated/システム共有/tm_afkinoDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             機能マスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_afkinoDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_afkino";
		public string kinoid { get; set; }                        //機能ID
		public string hyojinm { get; set; }                       //表示名称
		public string? programid { get; set; }                    //プログラムID（共用）
		public string? hanyokbn { get; set; }                     //汎用区分
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tm_afkinoDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             機能マスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tm_afkinoDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_afkino"
        Public Property kinoid As String                        '機能ID
        Public Property hyojinm As String                       '表示名称
        Public Property programid As String                    'プログラムID（共用）
        Public Property hanyokbn As String                     '汎用区分
    End Class
End Namespace
